src/client/css/fonts.css

```diff
@@ -197,6 +197,26 @@ https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap
 https://discourse.mozilla.org/t/metropolis-font-repo-missing/69415
 */
 
+@font-face {
+  font-family: Metropolis;
+  font-style: normal;
+  font-weight: 500;
+  font-display: optional;
+  src:
+    local('Metropolis-Medium'),
+    url('../fonts/Metropolis-Medium.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: Metropolis;
+  font-style: normal;
+  font-weight: 600;
+  font-display: optional;
+  src:
+    local('Metropolis-SemiBold'),
+    url('../fonts/Metropolis-SemiBold.woff2') format('woff2');
+}
+
```

Comment:
Sorry for the lack of context RE fonts:
It was discussed that we try to keep Refresh fonts to a minimum for simplicity and page-load considerations. Inter should have a regular and bold weight, Metropolis (which coincidentally is no longer supported) should have a bold weight. Please try to conform to these options and let UX know during review. If there is UX concern we can add new fonts.

Reply:
Addressed in https://github.com/mozilla/blurts-server/pull/2825#discussion_r1104363695.